test/JDBC/upgrade/latest/schedule

```diff
@@ -156,6 +156,7 @@ HAS_DBACCESS
 BABEL-1475
 BABEL-1510
 BABEL-3213
+BABEL-3010
```

Comment:
why only to latest schedule file? we have a minor version upgrade from 14.5 to latest views + catalog table, where do we cover it? also for system views which are upgraded, do we have existing test cases to cover them?

Reply:
I have only added BABEL-3010 to latest since the date columns are introduced in current release and they do not exists in previous releases. For system views, I think we already have test cases for them.

Reply:
sys.babelfish_view_def is part of 14.5, should have a minor version test from 14.5 to latest for this unless already covered. The test makes sure that existing rows for newly added columns have correct values

Reply:
I added a minor version upgrade test from 14.5 to latest, newly added columns create_date and modify_date will be null for existing rows which can be seen in the expected output file

Reply:
I already added major and minor version upgrade tests for 13.6 and 14.5 but the above approach looks better, so I have added the tests in the format given in the above PR.